### PR TITLE
Attempt to simplify/unify rules

### PR DESCRIPTION
**Pull request by russss**
As per #26 and #19. Comments welcome.

### CODE REVIEW

**Comment by urschrei** [comment]
I think

```
We think feminism is a good thing; we think brogramming is a bad thing. If you disagree, this may not be the place for you.
```

needs tweaking; I know that 'may' is British for 'is', but this is about concision and a clear statement of beliefs.

**Comment by russss** [comment]
Ah but now you're forcing me into saying saying brogramming is definitely a bad thing and how exactly do you want to define brogramming and argh :/

File: index.html [hunk 1]
```diff
@@ -40,29 +40,24 @@
...
-        <li>We think feminism is a good thing; we think brogramming is a bad thing.</li>
-        <li>We're a very informal and ad-hoc group right now, so if you piss us off you won't be invited again.</li>
+        <li>Don't harass people. Don't make exclusionary jokes. Don't even make them "ironically".</li>
+        <li>Discriminatory behaviour (such as homophobia, racism, transphobia, or sexism) is not appropriate.</li>
+        <li>We think feminism is a good thing; we think brogramming is a bad thing. If you disagree, this may not be the place for you.</li>
         <li>Although we're meeting in a pub, there is no expectation or pressure to drink alcohol. Don't question anyone's choice of drink.</li>
       </ul>
-      <p>The rules will become more explicit as we grow, and we're always trying to change for the better.</p>
+
+        <p>If you piss us off by breaking these rules, you won't be invited again.
```
Comment (ghost) [inline index.html:62]: IMHO I would s/If you piss us off by breaking/If you break/ - two thoughts:

1) This change might make it clearer that it's breaking the rules that will cause you to not be invited back, not breaking the rules to the point that we get mad (which I read as having a bit of wiggle room)
2) Obviously we want to sound serious here, but it reads to me a lot harsher than the rest of the computer-as-an-adjective text.  (edit: having thought about it for a moment, I think shortening the text has the side-effect of making it more forceful, which was the intent anyway.)

**Comment by urschrei** [comment]
I don't think it's incumbent upon you to define it! I also think that there's consensus about whether brogramming is a bad thing, and that Computer exists (in a vague way) in opposition to it.

**Comment by alexglow** [comment]
Hee! I like Computer <-> Brogramming dichotomy.

On Thu, Sep 26, 2013 at 5:26 PM, Stephan Hügel notifications@github.comwrote:

> I don't think it's incumbent upon you to define it! I also think that
> there's consensus about whether brogramming is a bad thing, and that
> Computer exists (in a vague way) in opposition to it.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/computeranonymous/computer/pull/39#issuecomment-25214529
> .

## 

Make your world! • http://alexglow.com
VSRE emails are cool. • http://vsre.info/

**Comment by russss** [comment]
Yeah so, as much as I love this joyously over-engineered pedantry (that wasn't sarcasm), my intention here is that we don't sacrifice the informal style in order to construct some cast-iron rules, at least to start with.

It is ultimately just a couple of people in the pub, and the aim of these rules is much more to put people at ease than it is to cope with the (vanishingly unlikely, IMO) case that someone wants to argue over wording after making an utter dick of themselves. Also, this isn't a democracy. We run this git repo, we can change it if there's a problem later on.

I'm going to defer to @tef as de-facto dictator and, since he's asleep, I'm going to do the same.

**Comment by ghost** [comment]
Fair enough - my intent was not for us to get caught up in language lawyering and miss the larger point :)

**Comment by ntlk** [comment]
IMHO it's a good idea to say outright that brogramming is A Bad Thing™ and that this _is not_ a place for you if you disagree. I think we should be clear that this group is a cultural opposite of everything brogramming vaguely means. That's A Good Thing™

File: index.html [hunk 2]
```diff
@@ -40,29 +40,24 @@
...
-
-      <p><a href="http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy">A standard code of conduct applies</a>. This means: Don't harass people. Don't make exclusionary jokes. Don't even make them "ironically".</p>
-
       <ul>
-        <li>We want to be inclusive; we don't welcome homophobic, racist, transphobic, or sexist behavior, amongst others.</li>
-        <li>We think feminism is a good thing; we think brogramming is a bad thing.</li>
-        <li>We're a very informal and ad-hoc group right now, so if you piss us off you won't be invited again.</li>
+        <li>Don't harass people. Don't make exclusionary jokes. Don't even make them "ironically".</li>
+        <li>Discriminatory behaviour (such as homophobia, racism, transphobia, or sexism) is not appropriate.</li>
+        <li>We think feminism is a good thing; we think brogramming is a bad thing. If you disagree, this may not be the place for you.</li>
```
Comment (DRMacIver) [inline index.html]: Out of curiosity, does "We think brogramming is a bad thing" really usefully exclude anyone that "We think feminism is a good thing" doesn't? If not, might it be worth removing due to ambiguity as to what it actually means?

Comment (DRMacIver) [inline index.html]: I also feel doing so meshes well with the "criticise with what people do, not what they are" attitude.

Comment (drtortoise) [inline index.html]: Some of my best friends are brogrammers. They're coming to the meetup.

Comment (finitud) [inline index.html]: This line is not against brogrammers (what people are), rather against brogramming (what people do). I'd say they're welcome as long as they follow the spirit of the group.

Moreover, I think it does add something useful. The group seems to be created, among others, along the lines of "we want a group that's not about brogrammers". It's more about creating a safe space for the people we want, and if the group attracts many brogrammers, some of the interesting discussions just won't happen at all.

File: index.html [hunk 3]
```diff
@@ -40,29 +40,24 @@
...
       <p>This is a support group. No-one knows what they are doing.</p>
-
-      <p>If you're worried about not being computer enough, come.</p>
-
-      <p>If your day job isn't code, come.</p>
-
-      <p>If you think you're an imposter, come.</p>
-
+      <p>If you're from a background which isn't well-represented in most geeky meetups, come along – we want to make a difference.</p>
+      <p>If you're worried about not being computer enough, come along.</p>
```
Comment (gearmonkey) [inline index.html:46]: Perhaps flip the order of L43 an L44? My thinking is that current L43 is basically a subset of the meaning of current L44 made more explicit, so I think the list might read better if the more general line is first.

File: index.html (hunk 2 above)
Comment (AgentAntelope) [inline index.html]: I agree with the second half of what @finitud has said here, but for my understanding of what a brogrammer is, I have no interest in making them feel welcome.

**Comment by russss** [comment]
I'm going to be slightly controversial and remove the "brogramming" reference.

It's clear from this thread that we're not particularly sure how to define "brogramming". I'm increasingly unsure that the people who we want to exclude would identify as "brogrammers" anyway.

I think the main thing we want to avoid from "brogrammers" is casual sexism. These rules already cover that.

File: index.html (hunk 2 above)
Comment (DRMacIver) [inline index.html]: I think the problem is that different peoples' understanding of "brogrammer" differs sufficiently that it doesn't make it clear _what_ we're excluding by excluding it.

**Comment by DRMacIver** [comment]
Perhaps what is needed/wanted to replace the brogrammer line is an explicit statement here that we actively want is for this not to be just a boys club. Maybe that's covered by avoiding sexism but I'm not sure it is.

**Comment by ntlk** [comment]
Also 'brogramming' has a whiff of "it's all about the tech" whereas this meetup is about people

**Comment by russss** [comment]
@DRMacIver I'd argue that's already pretty well covered in the "imposters welcome" section.

**Comment by AgentAntelope** [comment]
I disagree @russss, I'd like us to explicitly say we don't want this to be a boys club.

**Comment by DRMacIver** [comment]
I'd probably accept  your argument :-)

**Comment by AgentAntelope** [comment]
I don't believe the imposters welcome section mentions anything about us not being a boys club.

**Comment by ntlk** [comment]
Being an impostor is more about not entirely trusting your skills, I don't think it's explicitly a gendered thing, therefore I'd like to see it stated explicitly that diversity is not an afterthought.

**Comment by russss** [comment]
Argh I'm pretty sure this is scope creep. 

I'm not the official rules copywriter. If we can agree that this PR is a net improvement then we can merge it and then you're more than welcome to submit your own changes.

**Comment by DRMacIver** [comment]
I think @russss is talking about the line "If you're from a background which isn't well-represented in most geeky meetups, come along – we want to make a difference" this PR added. 

Could perhaps make the point more directly though.

**Comment by AgentAntelope** [comment]
I'd like that.

**Comment by public** [comment]
I think this is definitely an improvement. It would be nice to address specific groups of people we are trying to reach out to but that needs to happen carefully and I don't think it should block this PR.

**Comment by tef** [comment]
I'm /mostly/ happy with this commit. I understand why we don't want to use brogrammer/dudebro, but i think we're missing out in that "Boys will be boys" is not welcome, people are expected to be adults and treat each other with respect*
- Except this will come down to "BUT I'M JUST HAVING AN OPINON"

**Comment by public** [comment]
@tef I agree, and in the same way "Don't harass people" is really not an anti-harassment policy. I think we should expand on what sort of behaviour is not-ok in another PR though as this is still better than just linking to the GF wiki.

**Comment by tef** [comment]
If everyone thinks this commit is an improvement, let's merge it. We'll always be fixing the statement

**Comment by AgentAntelope** [comment]
I do not, as said in IRC.

**Comment by AgentAntelope** [comment]
I disagree with commit 7112839

**Comment by AgentAntelope** [comment]
In fact, with both commits.

**Comment by tef** [comment]
@AgentAntelope You're going to have to be explicit, i.e copy paste your irc comments in here :-)

**Comment by doismellburning** [comment]
Fwiw, glad to see brogramming clause has gone

**Comment by russss** [comment]
Pull request withdrawn. Hopefully it will provide some food for thought for the future, though.

**Comment by tef** [comment]
I think this is a good idea (Removing the link and writing our own policy in full), but I'm not sure we have a comprehensive enough policy to warrant it yet.

I think removing the link is the last thing we should do, and that for now we should focus on getting the code of conduct up to scratch. 

Then we'll make the one line change to remove the link/rephrase it

**Pull request closed without merging**